code/parachain/node/src/chain_spec/picasso.rs

```diff
@@ -25,6 +25,25 @@ pub fn genesis_config(
 	existential_deposit: Balance,
 	treasury: AccountId,
 ) -> picasso_runtime::GenesisConfig {
+	let mut contracts = Vec::new();
+	if let Some(contract) = include_contract(option_env!("CW_XC_GATEWAY_WASM_PATH")) {
+		contracts.push(contract);
+	}
+	if let Some(contract) = include_contract(option_env!("CW_XC_INTERPRETER_WASM_PATH")) {
+		// not sure what is going on, but it is weird
+		// Thread 'main' panicked at 'contracts in genesis are valid: Module(ModuleError { index:
+		// 180, error: [4, 0, 0, 0], message: Some("CodeValidation") })',
+		// /build/source/parachain/frame/cosmwasm/src/lib.rs:406 contracts.push(contract);
+	}
+
+	let cosmwasm = picasso_runtime::CosmwasmConfig {
+		contracts: contracts
+			.into_iter()
+			.map(|x| (root.clone(), x.try_into().expect("genesis cw wasm fits constraints")))
+			.collect(),
+		..Default::default()
+	};
```

Comment:
```suggestion
	if let Some(contract) = option_env!("CW_XC_GATEWAY_WASM_PATH") {
		contracts.push(contract);
	}
	if let Some(contract) = include_contract(option_env!("CW_XC_INTERPRETER_WASM_PATH")) {
		// not sure what is going on, but it is weird
		// Thread 'main' panicked at 'contracts in genesis are valid: Module(ModuleError { index:
		// 180, error: [4, 0, 0, 0], message: Some("CodeValidation") })',
		// /build/source/parachain/frame/cosmwasm/src/lib.rs:406 contracts.push(contract);
	}

	let contracts = contracts
		.into_iter()
		.map(|path| match std::fs::read(path).map(|bytes| bytes.try_into()) {
			Ok(Ok(data)) => contract,
			Ok(Err(err)) => panic!("{path}: {err}"),
			Err(err) => panic!("{path}: {err}"),
		})
		.map(|contract| (root.clone(), contract))
		.collect();

	let cosmwasm = picasso_runtime::CosmwasmConfig {
		contracts,
		..Default::default(),
	};
```

and then drop `include_contract` function.  There’s no need to open
and read the file by using `Read` trait there.  `std::fs::read` does
that for you.